app/data/pencil2d.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>pencil2d</id>
+  <launchable type="desktop-id">pencil2d.desktop</launchable>
```

Comment:
Food for thought: From the examples, it seems like some apps are using a reverse-DNS scheme for their desktop entry, too. Is there some sort of requirement or recommendation for this?